src/SUMMARY.md

```diff
@@ -62,7 +62,9 @@
   - [Named Structs](user-defined-types/named-structs.md)
   - [Tuple Structs](user-defined-types/tuple-structs.md)
   - [Enums](user-defined-types/enums.md)
-  - [Static and Const](user-defined-types/static-and-const.md)
+  - [Static and Const](user-defined-types/static.md)
+    - [Const](user-defined-types/const.md)
+    - [Properties](user-defined-types/properties.md)
```

Comment:
```suggestion
  - [Statics](user-defined-types/static.md)
  - [Constants](user-defined-types/const.md)
```

Per my suggestion, could you remove the properties slide and then remove the extra level of nesting here? The title of the first slide can also change to just "Statics" since it's not introducing both concepts anymore.